components/weather_data/WeatherStationDetail.tsx

```diff
@@ -225,7 +225,7 @@ export const WeatherStationDetail: React.FC<Props> = ({center_id, name, stations
     <VStack width="100%" height="100%" alignItems="stretch">
       <VStack width="100%" height="100%" p={16} space={8}>
         <HStack space={8} alignItems="center">
-          <BodyBlack>{name}</BodyBlack>
+          <BodyBlack style={{textTransform: 'capitalize'}}>{name.toLowerCase()}</BodyBlack>
```

Comment:
could just use lodash

```suggestion
          <BodyBlack>{_.capitalize(name)}</BodyBlack>
```

Reply:
We're doing this particular approach in quite a number of places today - I don't really understand why one would pick lodash over css here, or vice versa, and I'd rather keep it similar for now. Let's pull out an issue to follow up and fix them all if one approach is better than the other?

Reply:
it just looked weird to me that we're applying one string transform in JS and another in CSS, seems like it would be more succinct to have a a single JS call. but maybe I've done it before and started an unfortunate trend 🤷

Reply:
Yeah, makes sense. FWIW looking through the docs I think the two are not identical - `_.capitalize()` will not put into title case, just will capitalize the first word. No title case utility seems to exist in JavaScript, which is likely why we do this pattern?

Reply:
`/me shakes first at JS ecosystem`